app/views/course/assessment/questions/_question.html.slim

```diff
@@ -1,7 +1,9 @@
-= div_for(question, class: ['clearfix']) do
+= div_for(question, class: ['clearfix'], 'data-question-dom-id' => dom_id(question.acting_as)) do
   div.pull-right
     div.btn-group
       = edit_button(edit_path)
       = delete_button(delete_path)
-  h4 = format_inline_text(question.display_title)
+  h4
+    small => fa_icon 'arrows'.freeze
```

Comment:
Can we also have a message upon hover to prompt users that they are able to reorder by drag and drop? Not sure if it will cause trouble for touch devices.